style.css

```diff
@@ -8,6 +8,7 @@ body {
   font-family: 'Poppins', sans-serif;
   margin: 16px 20px;
   min-height: 100vh;
+  background-color: #ebebff;
 }
 
 header {
```

Comment:
[OPTIONAL]

Kindly consider adjusting your background image, it looks good, but it could look closer to the Figma design :wink: